7.workload_types.md

```diff
@@ -45,19 +48,55 @@ This section describes the different naming references for this kind.
 
 `singular` and `plural` can only contain characters from Unicode basic Latin 0030-0039 and 0061-007a.
 
-#### Setting
-
-Settings describe which pieces of a workload type are configurable.
-
-| Attribute | Type | Required | Default Value | Description |
-|-----------|------|----------|---------------|-------------|
-| `name` | `string` | Y | | The setting's name. Must be unique per workload type. |
-| `description` | `string` | N | | A description of the setting. |
-| `type` | `string` | Y | | The setting's type (One of `string`, `boolean`, `number`). |
-| `required` | `boolean` | N |`false` | Whether a value _must_ be provided for the setting. |
-| `default` | type indicated by `type` field | N | | The setting's default value. |
-
-The `name` field allows Unicode letters, numbers and `_` and `-`. The `description` field allows those characters plus whitespace and punctuation characters.
+### Properties
+
+A properties object is an object whose structure is determined by the workload type workloadSettings schema. It may be a simple value, or it may be a complex object.
```

Comment:
```suggestion
A properties is an object whose structure is determined by the workloadSettings schema. It may be a simple value, or a complex object.
```